Sources/OktaSQLiteStorage/README.md

```diff
@@ -1,9 +1,27 @@
 #  Okta SQLite Storage SDK 
 
-OktaSQLiteStorage is a lightweight wrapper on top of GRDB framework that allows easy db creation and handling migration scenarios
+OktaSQLiteStorage is a lightweight wrapper on top of GRDB framework that allows easy db creation and handling of migration scenarios
+
+## Getting started
+
+### CocoaPods
+
+This SDK is available through [CocoaPods](https://cocoapods.org). To install it, add the following line to your Podfile:
+
+```
+pod 'OktaSQLiteStorage'
+```
 
 ## Usage
 
+To start using sqlite storage you need to build and initialize `OktaSQLiteStorage` object
+Steps are the following:
+1. Create `SQLiteSchema` object with sql queries for your current db schema
+2. Use `SQLiteStorageBuilder` builder class to build an object that conforms to `OktaSQLiteStorageProtocol` protocol. Enable WAL mode and other settings if required
+3. Call `initialize` method to initialize `OktaSQLiteStorage` object and provide custom migrator object for migrating db schema. `OktaSQLiteStorage` can start migration process during this call and async call finishes whenever migration is fully completed or failed 
+4. You can now start read/write access to sqlite database. Use `sqlitePool` property for interacting with database via [GRDB](https://github.com/groue/GRDB.swift) intefaces
```

Comment:
Should you add an example of how to fetch a single record? I know it's a GRDB API but it took me a minute to realize this was the way to fetch/write data.